src/nimblepkg/options.nim

```diff
@@ -352,6 +352,11 @@ proc parseCmdLine*(): Options =
   if result.action.typ == actionNil and not result.showVersion:
     result.showHelp = true
 
+  # Inform user that we use their environment variables.
+  if result.getNimbleDir == getEnv("NIMBLE_DIR"):
+    display("Info:", "Using the 'NIMBLE_DIR' environment variable.",
```

Comment:
This will show the message even if ``NIMBLE_DIR`` is empty which isn't what we want IMO. Sorry, about the back and forth. I can fix this myself if you'd like.

Reply:
Really?  ``result.getNimbleDir`` will not return an empty string.

Reply:
oh, you're right. My bad.

Reply:
It's a bit chatty when you use it.  Perhaps we should implement something like ``go env``.

Reply:
oh, then you should make it a warning. IIRC Nimble keeps track of which warnings have been shown and ensures they are only shown once.

Reply:
alright, check #450